docs/DeveloperGuide.md

```diff
@@ -496,7 +520,8 @@ remove(key) | `removeRecipe(key)` | `removeIngredient(key)` | `removeConsumption
 </div>
 
 Given below is an example usage scenario and how the mechanism behaves:
-1. User inputs delete command with the item index, to delete that specific item from the respective list.
+1. User inputs delete command with the item index, to delete that specific item from their respective recipe
+ list, fridge or consumption list.
```

Comment:
i think i changed fridge to ingredient list to match the ug

Reply:
ok